src/components/ArticleMenu.js

```diff
@@ -51,46 +51,48 @@ export const ArticleMenu = ({
   }, [])
 
   const items = [
-    { title: i18n.i18n('search-placeholder'), action: onSearchSelected },
-    { title: i18n.i18n('article-action-sections'), action: onTocSelected },
-    { title: i18n.i18n('menu-textsize'), action: onTextsizeSelected }
-  ]
-
-  if (hasInfobox) {
-    items.push({
+    {
+      title: i18n.i18n('search-placeholder'),
+      action: onSearchSelected,
+      enabled: true
+    },
+    {
+      title: i18n.i18n('menu-previous'),
+      description: articleHistory.hasPrev() ? articleHistory.getPrev().title : null,
```

Comment:
Using ternary operator here to avoid this `TypeError` when there’s no previous article:

```
TypeError: _utils__WEBPACK_IMPORTED_MODULE_4__.articleHistory.getPrev(...) is undefined
```

Reply:
You don’t need the ternary here, just 

> articleHistory.getPrev().title